src/CLI/Program.cs

```diff
@@ -53,10 +53,15 @@ private int Run()
             Exception caughtException = null;
             try
             {
-                CaseInsensitiveParser().ParseArguments<Options>(_args)
+                using (var parser = CaseInsensitiveParser())
```

Comment:
>using [](start = 16, length = 5)

I wasn't sure about the lifetime here, since we use the helpWriter that we pass into the settings that the parser consumes. Looking at [https://github.com/commandlineparser/commandline/blob/master/src/CommandLine/ParserSettings.cs](https://github.com/commandlineparser/commandline/blob/master/src/CommandLine/ParserSettings.cs), though, it explicitly says that the helpWriter is ours.

Reply:
It makes sense that the Parser doesn't dispose of the HelpWriter because it doesn't allocate the TextWriter. I wouldn't expect this to change.